src/SFA.DAS.AssessorService.Web/Models/HomeIndexViewModel.cs

```diff
@@ -3,6 +3,6 @@
     public class HomeIndexViewModel
     {
         public bool UseGovSignIn { get; set; }
-        public string BannerViewPath => Constants.Banners.APARNotifyBannerViewPath;
+        public string BannerViewPath => Constants.Banners.GovNotifyBannerViewPath;
```

Comment:
If the APAR banner is going away - assuming that has been confirmed? - then get rid completly